x-pack/plugins/security_solution/public/management/components/endpoint_action_failure_message/endpoint_action_failure_message.test.tsx

```diff
@@ -102,7 +102,7 @@ describe('EndpointActionFailureMessage', () => {
           'agent-fails-a-lot': {
             type: 'json',
             content: {
-              code: 'ra_scan_error_scan-queue-quota',
+              code: 'ra_scan_error-queue-quota',
```

Comment:
based on endpoint code: https://github.com/elastic/endpoint-dev/blob/9db96d5c3319552b91513afc6e8eccffe78031d5/Python/endpoint/test/test_scan_response_actions.py#L336

this should be 
`ra_scan_error_queue-quota` (one more underscore instead of dash)

Reply:
Oh yeah! I updated it in d6384aec5652b83a58a70be5502bee7892859572